NitroxClient/Communication/MultiplayerSession/MultiplayerSessionManager.cs

```diff
@@ -78,6 +78,13 @@ public void RequestSessionReservation(PlayerSettings playerSettings, Authenticat
 
         public void ProcessReservationResponsePacket(MultiplayerSessionReservation reservation)
         {
+            if (reservation.ReservationState == MultiplayerSessionReservationState.ENQUEUED_IN_JOIN_QUEUE)
+            {
+                // TODO: Add this message in other languages
+                Log.InGame("Waiting in join queue...");
```

Comment:
```suggestion
              Log.InGame(Language.main.Get("Nitrox_Waiting"))
```
And you can simply add an entry `Nitrox_Waiting` to the english json in `Nitrox.Subnautica.Assets/LanguageFiles/English.json`